src/Docker.DotNet/Endpoints/ConfigsOperations.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Docker.DotNet.Models;
+
+namespace Docker.DotNet
+{
+    internal class ConfigsOperations : IConfigsOperations
```

Comment:
Don't we need tests for this class too?

Reply:
Yes, I'll add them.

Reply:
Added: Create, List, Inspect and Remove tests.